lux-cli/src/debug.rs

```diff
@@ -2,7 +2,20 @@ use crate::{
     project::DebugProject,
     unpack::{Unpack, UnpackRemote},
 };
-use clap::Subcommand;
+use clap::{Args, Subcommand};
+
+#[derive(Args)]
+pub struct Dependencies {
+    /// Output format: human-readable or json
```

Comment:
issue: No need to specify "human-readable or json" here.
Clap will generate help for the enum variants.

```suggestion
    /// The output format.
```

results in the following CLI `--help` message:

```
--output-format <OUTPUT_FORMAT>  The output format [default: text] [possible values: json, text]
help
```